core/web/user_controller_test.go

```diff
@@ -17,7 +18,7 @@ import (
 
 func TestUserController_UpdatePassword(t *testing.T) {
 	app := cltest.NewApplicationEVMDisabled(t)
-	require.NoError(t, app.Start())
+	require.NoError(t, app.Start(context.TODO()))
```

Comment:
Perhaps we could use a helper which combines:
https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/terra/terratxm/txm_internal_test.go#L228-L236
with
https://github.com/smartcontractkit/chainlink/blob/develop/core/internal/cltest/cltest.go#L940-L946
to drop in to tests?
`cltest.TestContext(*testing.T)`, or maybe a lighter package than `cltest` that can be imported from anywhere :shrug:

Reply:
I'm also second guessing that 90% scale now - maybe subtracting some fixed amount of time would be better? e.g. `5s`

Reply:
Separate PR here: https://github.com/smartcontractkit/chainlink/pull/5993